sqlserver/datadog_checks/sqlserver/connection.py

```diff
@@ -559,7 +586,10 @@ def test_network_connectivity(self):
         """
         host, port = split_sqlserver_host_port(self.instance.get('host'))
         if port is None:
-            port = 1433
+            port = DEFAULT_CONN_PORT
```

Comment:
should this line 
```
host, port = split_sqlserver_host_port(self.instance.get('host'))
```

simply just call this new function you added? `_get_host_with_port` then we wouldn't need any of the port validation below right?

Reply:
Since this function is designed to test the config settings and return a specific error message about anything malformed, i think we should leave this as-is.

Reply:
oh i didnt noticed this line https://github.com/DataDog/integrations-core/pull/12610/files#diff-a38a2a09e86a943d0704a9f2922133905324820865ff052eff46209ee989d6d9R597, okay thats fine 👍